Source/WebCore/dom/Position.cpp

```diff
@@ -337,12 +337,12 @@ Position::AnchorType Position::anchorTypeForLegacyEditingPosition(Node* anchorNo
 }
 
 // FIXME: This method is confusing (does it return anchorNode() or containerNode()?) and should be renamed or removed
-Element* Position::element() const
+RefPtr<Element> Position::anchorElementAncestor() const
```

Comment:
This is fine but personally I hadn't been converting functions like these. It's all simple getters so it should be safe without smart pointers.

Reply:
Alright, I'm gonna keep this change for now.